docs/conf.py

```diff
@@ -1,3 +1,5 @@
+from recommonmark.parser import CommonMarkParser
```

Comment:
I think this module needs to be vendored. (from https://github.com/rtfd/recommonmark).
Also, it needs to be added to the setup.py file.

Reply:
this PR is only touching the `docs/` folder, so it's separate from the `cli/` folder